manifests/params.pp

```diff
@@ -84,10 +84,10 @@
 
       if pick($service_provider, $facts['service_provider']) == 'systemd' {
         $service_reload = "systemctl reload ${service_name}"
-        $service_status = "systemctl status ${service_name}"
+        $service_status = pick($service_status, "systemctl status ${service_name}")
```

Comment:
See https://github.com/puppetlabs/puppetlabs-postgresql/pull/1349 for more discussion on that. I also wonder if this isn't really redundant since AFAIK Puppet itself can determine the status using these commands.